lang/package.go

```diff
@@ -124,6 +145,16 @@ func (pkg *Package) Name() string {
 	return pkg.doc.Name
 }
 
+// Title provides the formatted name of the package. It is primarily designed for
+// generating headers.
```

Comment:
Headers are a markdown thing that can take different levels of document headers. Do you mean the file header text provided by `--header`? We should avoid unnecessary ambiguity.